### PR TITLE
Migrate to GO

### PR DESCRIPTION
**Pull request by pratishshr**
- Rewrite entire codebase in go

### CODE REVIEW

File: platforms/aws/aws.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package aws
+
+import (
+	"encoding/json"
```
Comment (leapfrog-saugat) [inline platforms/aws/aws.go:4]: Use bharyang :D

Comment (pratishshr) [reply platforms/aws/aws.go:4]: Go formatter uses ascending order for imports  ;P

Comment (leapfrog-saugat) [reply platforms/aws/aws.go:4]: Tara looks like first three are not following the format.

Comment (pratishshr) [reply platforms/aws/aws.go:4]: ascending order by name, not by length as bharyang

Comment (leapfrog-saugat) [reply platforms/aws/aws.go:4]: eh hehe.. did'nt understand the first time :D

**Pull request merged**